### DISCUSSION TITLE
How to control rendering order of content item parts?

### DISCUSSION THREAD

**Discussion by xperiandri** [OrchardCMS/OrchardCore#8960] [Q&A]
We use `DisplayAsync` to render a whole content item, but the order does not correspond to the order in a content type.
How to control that?

**Comment by deanmarcussen** [comment]
Placement

https://docs.orchardcore.net/en/dev/docs/reference/core/Placement/#format

```
{
  "TextField": [
    {
      "displayType": "Detail",
      "differentiator": "Article-MyTextField",
      "place": "Content:20", << adjust order here. valids are `Content:x.y`, `Content:before`, `Content:after` 
    }
  ],
}
```

**Reply by xperiandri** [reply]
Is it some how adjustable through admin UI?

**Reply by Skrypt** [reply]
Enable Placement feature and you can add placements from the admin UI.
